app/javascript/vue/shared/content/attachments/list.vue

```diff
@@ -60,12 +102,21 @@ export default {
   },
   data() {
     return {
-      imageLoadError: false
+      imageLoadError: false,
+      hovered: false
     };
   },
   methods: {
     handleImageError() {
       this.imageLoadError = true;
+    },
+    handleMenuVisibilityChange(isOpen) {
+      const { listAttachmentContainer } = this.$refs;
+      if (isOpen) {
+        listAttachmentContainer.classList.add('context-menu-open');
```

Comment:
This should be done using a "this.menuOpen" reactive data property, and a conditional class assignment in the template :class="{ 'context-menu-open': menuOpen }"